demo/qat-init.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 # This script is based on qatlib's qat_init.sh
 NODE_NAME="${NODE_NAME:-}"
-ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-37c8 4940 4942}
+ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-37c8 4940 4942 4944}
```

Comment:
Out of curiosity, why is the indexing different (-1) here vs. the go code?

Reply:
Maybe..
here: PF
dpdkdrv: VF ?

Reply:
@hj-johannes-lee probably remembers